common/rdr/FileInStream.cxx

```diff
@@ -56,7 +56,7 @@ int FileInStream::pos()
   return ftell(file) + ptr - b;
 }
 
-int FileInStream::overrun(int itemSize, int nItems, bool wait)
+int FileInStream::overrun(int itemSize, int nItems, bool /* wait */)
```

Comment:
Updated patchset only touches PixelBuffer.h, InputHandler.h, and SDesktop.h in a much nicer way.